pkg/kubelet/logs/container_log_manager.go

```diff
@@ -171,20 +173,28 @@ func NewContainerLogManager(runtimeService internalapi.RuntimeService, osInterfa
 			MaxSize:  parsedMaxSize,
 			MaxFiles: maxFiles,
 		},
-		clock: clock.RealClock{},
-		mutex: sync.Mutex{},
+		clock:            clock.RealClock{},
+		mutex:            sync.Mutex{},
+		maxWorkers:       maxWorkers,
+		queue:            workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "kubelet_log_rotate_manager"),
+		monitoringPeriod: monitorDuration,
 	}, nil
 }
 
 // Start the container log manager.
 func (c *containerLogManager) Start() {
 	ctx := context.Background()
+	klog.InfoS("Initializing container log rotate workers", "workers", c.maxWorkers, "monitorPeriod", c.monitoringPeriod)
```

Comment:
Do we want this always logging?

Maybe we add a V(4) here like the rest of the code?

Reply:
follow up is fine. no need to rev this again unless we are fixing something else